install.sh

```diff
@@ -9,77 +17,95 @@ log () {
 
 # Root check
 if [ $EUID -ne 0 ]; then
-  echo "Du musst dieses Skript als root ausführen"
+  echo -e "$RED✗ Fehler bei der Installation:$NORMAL Du benötigst Root-Rechte, um die Installation zu starten."
   exit
 fi
 
 # Check if installed
 if [ -d $INSTALLATION_PATH ]; then
-    log "Eine MySpeed-Instanz unter $INSTALLATION_PATH wurde bereits installiert. Die Installation wird abgebrochen."
+    echo -e "$YELLOW⚠ Fehler bei der Installation: $NORMAL MySpeed ist bereits auf diesem System installiert. (Pfad: $INSTALLATION_PATH)"
     exit 0
 fi
 
 
 # Update all packages
+echo -e "$BLUE🔎 Status:$NORMAL Es wird nach neuen Updates für das Linux-System gesucht..."
 apt-get update -y
 
+clear
+echo -e "$GREENℹ Info:$NORMAL Die Installation wird jetzt vorbereitet. Das kann einen Augenblick dauern..."
+sleep 5
 # Check for wget
+clear
+echo -e "$BLUE🔎 Status:$NORMAL Überprüfe, ob wget vorhanden ist..."
 if ! command -v wget &> /dev/null
 then
-    log "Das Paket \"wget\" wurde nicht gefunden, wird aber benötigt. Es wird nun installiert..."
+    echo -e "$YELLOWℹ\"wget\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
     sleep 2
     apt-get install wget -y
 fi
 
 # Check for unzip
+clear
+echo -e "$BLUE🔎 Status:$NORMAL Überprüfe, ob unzip vorhanden ist..."
 if ! command -v unzip &> /dev/null
 then
-    log "Das Paket \"unzip\" wurde nicht gefunden, wird aber benötigt. Es wird nun installiert..."
+    echo -e "$YELLOWℹ\"unzip\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
     sleep 2
     apt-get install unzip -y
 fi
 
 # Check for curl
+clear
+echo -e "$BLUE🔎 Status:$NORMAL Überprüfe, ob curl vorhanden ist..."
 if ! command -v curl &> /dev/null
 then
-    log "Das Paket \"curl\" wurde nicht gefunden, wird aber benötigt. Es wird nun installiert..."
+    echo -e "$YELLOWℹ\"curl\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
```

Comment:
Hier sollte noch ein Leerzeichen ergänzt werden, weil sonst die Anführungszeichen direkt neben dem Emoji stehen, was ein bisschen komisch aussieht 😅
```suggestion
    echo -e "$YELLOWℹ \"curl\" ist nicht installiert.$NORMAL Die Installation wurde gestartet..."
```